pkgs/applications/misc/brewtarget/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib,
+mkDerivation,
```

Comment:
I always find the way the qt manual wants to suggest this style not great.

```
{ lib
, mkDerivation
, ...
```

is what's already suggested in the "style guide".